src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryConfig.java

```diff
@@ -149,6 +149,12 @@
                      " and on the alias update time if the number of aliases is huge (over 10000).")
     boolean resource_resolver_optimize_alias_resolution() default true;
 
+    @AttributeDefinition(name = "Allowed Optimize alias path",
```

Comment:
Nit: I think "Allowed optimized alias paths" communicated the intent better.

Reply:
Or similar to the vanity equivalent "Allowed Alias Locations".